src/cobalt.rs

```diff
@@ -53,11 +53,11 @@ pub fn build(config: &Config) -> Result<()> {
         .map(OsStr::new)
         .collect();
 
-    let layouts = source.join(&config.layouts);
-    let mut layouts_cache = HashMap::new();
+    let templates_dir = source.join(&config.templates);
+    let mut templates_cache_dir = HashMap::new();
```

Comment:
I renamed this variable from `layouts_cache` to `templates_cache_dir`, because, canonically, it is the "cache directory for templates". Although if you think `templates_cache` makes more sense and is more in line with the rest of the codebase, this is obviously a simple thing to change.